### DISCUSSION TITLE
Error Response modification and XML to JSON format

### DISCUSSION THREAD

**Discussion by vubon** [minio/minio#12983] [Q&A] [answered]
Hi guys,
Is there any way to change the error message and success message for presigned URL response? Currently, it is always responding to XML format. How can I make the JSON format? Please suggest me. Thanks in advance

**Comment by harshavardhana** [comment, marked as answer]
Error response in XML is AWS S3 API requirement, it can't be changed to JSON

**Reply by vubon** [reply]
Thank you so much for your reply.

**Reply by huggingfaceme** [reply]
when amazon does sh!@#!3t, you shouldn't follow their rule... thx for your 1 minute reply
